### PR TITLE
docs: AWS CDK comparison

### PR DESCRIPTION
**Pull request by davemooreuws**

### CODE REVIEW

**Comment by tjholm** [comment]
Let's remove incomplete pages to be introduced in new branches? We can introduce new alternatives pages progressively rather than blocking on one PR.

File: src/pages/faq/comparison/terraform.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+resource "aws_lambda_permission" "api_gw" {
+  statement_id  = "AllowExecutionFromAPIGateway"
+  action        = "lambda:InvokeFunction"
+  function_name = aws_lambda_function.hello_world.function_name
+  principal     = "apigateway.amazonaws.com"
+
+  source_arn = "${aws_apigatewayv2_api.lambda.execution_arn}/*/*"
+}
+
+```
```
Comment (tjholm) [inline src/pages/faq/comparison/terraform.mdx:179]: We also need to include runtime code here for the terraform example as well, can the component do this?

Comment (davemooreuws) [reply src/pages/faq/comparison/terraform.mdx:179]: All the tf config all needs to be replaced. The PR is still a draft WIP

**Comment by davemooreuws** [comment]
Note: Allow for grouping (e.g. 3 files)

File: src/pages/faq/comparison/aws-cdk.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+| ---------------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| Approach                           | Infrastructure from code                                                                | Infrastructure as code                                                                                  |
+| Language                           | Your choice                                                                             | Your choice                                                                                             |
+| Cloud Mechanics (IAM, Networking)  | Generated automatically from intent                                                     | Manual                                                                                                  |
+| Extensibility                      | Custom resources                                                                        | Custom Constructs / Resource Providers                                                                  |
```
Comment (tjholm) [inline src/pages/faq/comparison/aws-cdk.mdx:81]: Would be good to clarify what we mean by custom resources.

**Comment by HomelessDinosaur** [comment]
The vertical centring of the text in the FAQ squares is slightly off.

**Review by jyecusch — DISMISSED** [review]

File: src/pages/faq.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+export const disableEditGithub = true
+
+# Nitric Frequently Asked Questions (FAQ)
+
+<FAQGrid />
+
+## More questions?
+
+Come and chat on [GitHub](https://github.com/nitrictech/nitric/discussions).
```
Comment (HomelessDinosaur) [inline src/pages/faq.mdx:12]:
```suggestion
Start a discussion on [GitHub](https://github.com/nitrictech/nitric/discussions) or come and chat on our community [Discord](https://discord.com/invite/Webemece5C).
```

**Review by tjholm — APPROVED** [review]

**Pull request merged**